app/views/aid_applications/duplicates/show.html.erb

```diff
@@ -8,27 +8,50 @@
     <div class="form-card__content">
       <p><%= t('.body') %></p>
     </div>
+    <div class="data-table-container">
+      <table class="data-table js-sticky-header">
+        <thead>
+          <tr> <!-- empty header renders top line on table-->
+            <th></th>
+            <th></th>
+          </tr>
+        </thead>
+        <tbody>
+          <% AidApplication.matching_submitted_apps(@aid_application).each do |aid_application| %>
+            <%= render 'data_row', header: 'Name', value: @aid_application.name == aid_application.name ? 'Same' : 'Similar', aid_application: aid_application %>
+            <%= render 'data_row', header: 'DOB', value: 'Same', aid_application: aid_application %>
+            <%= render 'data_row', header: 'Address', value: @aid_application.street_address == aid_application.street_address ? 'Same' : 'Similar', aid_application: aid_application %>
+            <%= render 'data_row', header: 'Phone number', value: aid_application.phone_number, aid_application: aid_application %>
+            <%= render 'data_row', header: 'Date submitted', value: aid_application.submitted_at.strftime("%d-%m-%Y"), aid_application: aid_application %>
+            <%= render 'data_row', header: 'Worker', value: aid_application.submitter.name, aid_application: aid_application %>
+            <%= render 'data_row', header: 'CBO', value: aid_application.organization.name, aid_application: aid_application %>
+            <%= render 'data_row', header: 'Case status', value: aid_application.status.capitalize, aid_application: aid_application %>
+        <% end %>
+        </tbody>
+      </table>
+    </div>
     <div class="form-card__footer">
       <div class="toolbar">
         <div class="toolbar__left">
-          <%
-            go_back_path = if verify_step?
-                             edit_organization_aid_application_verification_path(current_organization, current_aid_application)
-                           else
-                             edit_organization_aid_application_applicant_path(current_organization, current_aid_application)
-                           end
-          %>
-          <%= link_to go_back_path, class: 'button button--primary' do %>
-            <span class="icon icon-arrow_back" aria-hidden="true"></span> <%= t('general.go_back') %>
-          <% end %>
-        </div>
-        <div class="toolbar__right">
           <%= button_to t('general.delete_this_application'),
                         organization_aid_application_path(current_organization, current_aid_application),
                         method: :delete,
                         class: "button button--danger",
                         data: { confirm: "Are you sure you want to delete this application?" } %>
         </div>
+        <div class="toolbar__right">
+        <% if verify_step? %>
```

Comment:
The `verify_step?` parameter (`?verify=true`) was previously used for a cosmetic link, but now it's being used to expose some action buttons. There should be some further verification in the controller that the application is in an appropriate state and not wholly trust the url.